west.yml

```diff
@@ -214,7 +214,7 @@ manifest:
         - hal
     - name: hal_renesas
       path: modules/hal/renesas
-      revision: af77d7cdfeeff290593e7e99f54f0c1e2a3f91e6
+      revision: 24543f3187ed90c4556466d0fe6b4cd948140648
```

Comment:
Better use the latest hash.

Reply:
Update done.

Reply:
HAL was updated yesterday. Please update it one more.

Reply:
Update done.